bHY6yh-de4030/cod-vyg67d_bHY6yh-de4030.xml

```diff
@@ -0,0 +1,285 @@
+            </p>
+            <p xml:id="bHY6yh-de4030-d1e293">
+               <ref xml:id="bHY6yh-de4030-Rd1e330">SAlomon in prouerb. capitulo. 15.</ref> ¶ <quote xml:id="bHY6yh-de4030-Qd1e297">Longe est 
+               <lb ed="#O"/>dominus ab impiis: et orationes iustorum 
+               <pb ed="#O" n="15-v"/>
+               <cb ed="#O" n="a"/>
+               <lb ed="#O"/>exaudiet.
+               </quote>
+            </p>
+            <p xml:id="bHY6yh-de4030-d1e307">
+               <!--line bumps down-->
+               <ref xml:id="bHY6yh-de4030-Rd1e346">Tobie hy<!--line bumps up-->storia capitulo 4.</ref> ¶ <quote xml:id="bHY6yh-de4030-Qd1e311">Omnitempore bene<!--line bumps back down-->dic deum et pete ab eo ut 
+               <lb ed="#O"/>uias tuas dirigat: et omnia con<lb ed="#O" break="no"/>silia
+               tua in ipso permaneant.</quote>
+            </p>
+            <p xml:id="bHY6yh-de4030-d1e319">
+               ¶ <ref xml:id="bHY6yh-de4030-Rd1e358">12. capitulo.</ref> <quote xml:id="bHY6yh-de4030-Qd1e321">
+               <lb ed="#O"/>bona est oratio cum ieiunio et helimosi<lb ed="#O" break="no"/>na 
+               magis quam thesauros auri con<lb ed="#O" break="no"/>dere.</quote>
+            </p>
+            <p xml:id="bHY6yh-de4030-d1e329">
+               <!--line bumps down-->
+               <ref xml:id="bHY6yh-de4030-Rd1e372">Ie<!--line bumps up-->sus filius sirach in ecclesiastico. capitulo. 3.</ref> ¶ <quote xml:id="bHY6yh-de4030-Qd1e333">Qui <!--line bumps back down--> diligit deum exorabit pro pecca<lb ed="#O" break="no"/>tis
+               et continens est ab illis et in 
+               <lb ed="#O"/>oratione dierum exaudietur.</quote>
+            </p>
+            <p xml:id="bHY6yh-de4030-d1e342">
+               ¶ <ref xml:id="bHY6yh-de4030-Rd1e385">18 capitulo.</ref>
+               <lb ed="#O"/><quote xml:id="bHY6yh-de4030-Qd1e344">ante orationem praepara animam tuam et noli
+               <lb ed="#O"/>esse quasi homo qui temptat deum.</quote>
+            </p>
+            <p xml:id="bHY6yh-de4030-d1e352">
+               <lb ed="#O"/><ref xml:id="bHY6yh-de4030-Rd1e399">Mattheus in euangelio. 6. capitulo.</ref> ¶ <quote xml:id="bHY6yh-de4030-Qd1e356">Cum oratis 
+               <lb ed="#O"/>non eritis sicut hypocrite qui amant 
+               <lb ed="#O"/>in synagogis et angulis platearum 
+               <lb ed="#O"/>stantes orare vt videantur ab hominibus 
+               <lb ed="#O"/> ¶ Intra cubiculum tuum et clauso ho<lb ed="#O" break="no"/>stio 
+               tuo ora patrem tuum in abscondi<lb ed="#O" break="no"/>to
+               reddet tibi et pater tuus qui ui<lb ed="#O" break="no"/>det 
+               in abscondito reddet tibi. o<lb ed="#O" break="no"/>rantes 
+               nolite multum loqui. ¶ Sic 
+               orabitis: Pater noster qui es in <sic>celis</sic> sanctifice<lb ed="#O" break="no"/>totur 
+               [omitted]
```

Comment:
If only half of the paragraph matches add the note (which you did and it great) but you don't need to add "omitted"